config/routes.rb

```diff
@@ -1,5 +1,6 @@
 Rails.application.routes.draw do
   devise_for :users
   root 'welcome#index'
+  resources :cv_user_main_details, only: [:new]
```

Comment:
@AkshayPashnathSuresh  
you havent changed the controller name yet. 
as we discussed earlier, change it to `personal_info` or something easy to to understand. 
`cv_user_main_details` is a blackbox and too long